doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1634,50 +1709,50 @@ OCT-2587: A CIS gateway will try to connect to a BIS gateway
 
   **Workaround:** Use different :kconfig:option:`CONFIG_BT_DEVICE_NAME` for BIS and CIS.
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCT-2712: Gateway cannot properly handle a connected headset if it continues to advertise
   The gateway will try to connect to a headset that is already connected.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCT-2713: Gateway can only read one PAC record from each headset
   Some headsets might have several PAC records containing different supported configurations.
   Then the most optimal configuration might not be used, or the gateway might not be able to configure a headset since it does not know about all supported configurations.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCT-2715: There can be a state mismatch between streaming and media control in certain scenarios
   For example, if a stream is in the playing state but media control is in the pause state, it might not be possible to play or pause.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0 v2-4-3 v2-4-2 v2-4-1 v2-4-0
 
 OCT-2725: Data can be overwritten when publishing configurations too fast on :ref:`zephyr:zbus`
   When, for example, setting up a bidirectional stream, the configurations for sink and source might be received too fast, resulting in one of the configurations being lost.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCT-2765: BIS headset stack overflow if gateway is periodically reset
   A BIS headset might get stack overflow if the BIS source is reset several times with short intervals.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
 OCT-2766: BIS headset might get into an infinite loop
   If a BIS headset is paused, the BIS broadcaster is reset, and if the headset is unpaused, it will end up in an infinite loop.
 
   **Affected platforms:** nRF5340 Audio DK
 
-.. rst-class:: v2-5-2 v2-5-1 v2-5-0
+.. rst-class:: v2-6-0 v2-5-2 v2-5-1 v2-5-0
 
```

Comment:
Please add:

OCT-2172:
The headset volume is not stored persistently. This means the volume will fall back to default level after a reset.

Reply:
Only for v2-6-0?

Reply:
It has been for earlier releases as well, but ATM we don't know when this issue was introduced.